src/commonMain/kotlin/fr/acinq/lightning/wire/InitTlv.kt

```diff
@@ -31,4 +32,22 @@ sealed class InitTlv : Tlv {
             }
         }
     }
+
+    @Serializable
+    data class PhoenixAndroidLegacyNodeId(@Contextual val legacyNodeId: PublicKey) : InitTlv() {
+        override val tag: Long get() = Networks.tag
```

Comment:
```suggestion
        override val tag: Long get() = PhoenixAndroidLegacyNodeId.tag
```

Reply:
How is the unit test passing...??

Reply:
See https://github.com/ACINQ/lightning-kmp/pull/319#discussion_r798327722